internal_tests/pytests/grid_stat/test_grid_stat_wrapper.py

```diff
@@ -120,6 +120,172 @@
 
         ({'GRID_STAT_OUTPUT_PREFIX': 'my_output_prefix'},
          {'METPLUS_OUTPUT_PREFIX': 'output_prefix = "my_output_prefix";'}),
+
+        ({'GRID_STAT_OUTPUT_FLAG_FHO': 'STAT', },
```

Comment:
@georgemccabe does the 'STAT' here mean that by default all of these output types are turned on? In the default Grid-Stat config file, they are all turned off by setting them to 'NONE':
https://github.com/dtcenter/MET/blob/5b3d8f7a15879bc49e3a0ebf8b75672da4568327/met/data/config/GridStatConfig_default#L210

So the user must actually decide what sort of output they want.

Reply:
The default behavior if none of the GRID_STAT_OUTPUT_FLAG_* variables are set is to set METPLUS_OUTPUT_FLAG_DICT to an empty string. This results in all of the default values from the share/met/config file. If a single flag is set in the METplus config, only that value overrides the default value and the rest of the defaults are still used.